profiles/2017-03-09/definition.json

```diff
@@ -21,5 +21,8 @@
         "../../services/resources/mgmt/2016-02-01/resources/resourcesapi",
         "../../services/resources/mgmt/2016-06-01/subscriptions",
         "../../services/resources/mgmt/2016-06-01/subscriptions/subscriptionsapi"
-    ]
+    ],
+    "outDir": {
+        "github.com/Azure/azure-sdk-for-go/services/preview/resources/mgmt/2015-10-01-preview/policy": "resources/mgmt/policy"
```

Comment:
nit: should we call this more like pathOverride/aliasPathOverride or something different from outDir? seems less coupled

Reply:
Was thinking about that, I like `pathOverride`.